async/dune

```diff
@@ -1,24 +1,20 @@
 (* -*- tuareg -*- *)
-#require "unix"
 
 let coverage_rewriter =
-  let is_coverage = try Unix.getenv "BISECT_ENABLE" = "YES" with Not_found -> false in
-  if is_coverage then
-    "(preprocess (pps (bisect_ppx -conditional)))"
-  else
-""
+  let bisect = Jbuild_plugin.V1.run_and_read_lines "echo ${BISECT_ENABLE}"
```

Comment:
You can use `let bisect = try Sys.getenv "BISECT_ENABLE" with _ -> ""` here if you want to avoid spawning a shell.